public/sass/components/_dashboard_grid.scss

```diff
@@ -85,6 +85,15 @@
   }
 }
 
+// Hack to prevent panel overlap during drag/hover (due to descending z-index assignment)
+.react-grid-item {
+  &:hover,
+  &:active,
+  &:focus {
+    z-index: 100 !important;
```

Comment:
This will not work as expected on dashboard with more than 100 panels. The top panels would have z-index 300, 299, 298, 297, etc. (for example), and if the one we're hovering on is any of the panels which are not the last 100, then the regression mentioned in the issue is back.

Reply:
right, i figured a dashboard with 100 panels would be unusual enough where we can wait for a bug report and then bump it to 200 or something more. i dont see a cheap/simple solution that does not hardcode _some_ number. i really want to avoid doing this on JS and probing the DOM for the first panel's z-index to find the exact limit.

Reply:
i guess an alternative could be to inject these hover/active styles golbally by the dashboard, but let's see if we can get away with this simple version first ;)

Reply:
I see your point.
It doesn't have to check anything if it just uses the panels length where it already has access to it (`const descIndex = this.props.dashboard.panels.length - panelElements.length;`). But maybe that's what you mean by "i guess an alternative could be to inject these hover/active styles golbally by the dashboard". 

I think cases with over 100 panels are more than common, but let's see how it turns out indeed.